int16 count
MagMeasurement[] samples
